README.md

```diff
@@ -18,56 +18,83 @@ To install the latest released version of the library, we recommend using [conda
 and the [astroconda](https://astroconda.readthedocs.io) channel to install the
 software and using a conda environment for the installation:
 
-    conda create -n jwst --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults python=3 jwst
-    source activate jwst
+    conda create -n jwst --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults python=3.6 jwst
+    conda activate jwst
```

Comment:
Since when did the "source" command change to "conda"? Do both still work?

Reply:
It changed in the last few months.  I believe both still work.  If you have python 3.6 and the latest conda though, `conda` works.  I can back out this change if you prefer.

Reply:
Ah, ok.  I will change it back.